hack/local-up-cluster.sh

```diff
@@ -818,8 +818,12 @@ function create_storage_class {
 
 function print_success {
 if [[ "${START_MODE}" != "kubeletonly" ]]; then
+  echo -n "Local Kubernetes cluster is running."
+  if [[ "${ENABLE_DAEMON}" = false ]]; then
```

Comment:
Actually, can you swap this to the if ... else logic you have below? The -n's "work", but also there is no space between the "is running.Please Ctrl-C" bit. The if..else logic is easier to read too.